main.py

```diff
@@ -1,7 +1,35 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Genre, Actor
 
 
 def main() -> QuerySet:
-    pass
+    genres = ["Western", "Action", "Dramma"]
+    actors = [
+        ("George", "Klooney"),
+        ("Kianu", "Reaves"),
+        ("Scarlett", "Keegan"),
+        ("Will", "Smith"),
+        ("Jaden", "Smith"),
+        ("Scarlett", "Johansson"),
+    ]
+
+    for genre in genres:
+        Genre.objects.create(name=genre)
+    for first, last in actors:
+        Actor.objects.create(
+            first_name=first,
+            last_name=last
+        )
+
+    Genre.objects.filter(name="Dramma").update(name="Drama")
+    Actor.objects.filter(last_name="Klooney").update(last_name="Clooney")
+    Actor.objects.filter(last_name="Reaves").update(
+        first_name="Keanu", last_name="Reeves"
+    )
+
+    Genre.objects.filter(name="Action").delete()
+    Actor.objects.filter(first_name="Scarlett").delete()
+
+    return Actor.objects.filter(last_name="Smith").order_by("first_name")
```

Comment:
It seems that you forgot to add migrations to your PR. Please, check the fourth point of the checklist.